core/types/bid.go

```diff
@@ -0,0 +1,44 @@
+package types
+
+import (
+	"math/big"
+	"sync/atomic"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+// TODO(roshan) refer to validator for all not in miner
+
+// Bid represents a bid.
+type Bid struct {
+	BlockNumber uint64          `json:"blockNumber"`
+	ParentHash  common.Hash     `json:"parentHash"`
+	Txs         []hexutil.Bytes `json:"txs,omitempty"`
+	GasUsed     uint64          `json:"gasUsed"`
+	GasFee      uint64          `json:"gasFee"`
```

Comment:
`GasFee` should be `big.Int`, the validator repo already change this. Keep the same, other wise, if the `GasFee` is over 10BNB, uint64 can not represent it.

Reply:
Done